extensions/sql-migration/src/wizard/accountsSelectionPage.ts

```diff
@@ -80,6 +97,50 @@ export class AccountsSelectionPage extends MigrationWizardPage {
 		};
 	}
 
+	private createAzureTenantContainer(view: azdata.ModelView): azdata.FormComponent {
+
+		const azureTenantDropdownLabel = view.modelBuilder.text().withProps({
+			value: constants.AZURE_TENANT,
+			CSSStyles: {
+				'margin': '0px'
+			}
+		}).component();
+
+		this._accountTenantDropdown = view.modelBuilder.dropDown().withProps({
+			width: WIZARD_INPUT_COMPONENT_WIDTH
+		}).component();
+
+		this._accountTenantDropdown.onValueChanged(value => {
+			/**
+			 * Replacing all the tenants in azure account with the tenant user has selected.
+			 * All azure requests will only run on this tenant from now on
```

Comment:
Instead of doing this why wouldn't we just modify the azurecore functions to take in an optional list of tenants? That's going to be much easier and straightforward to use than expecting people to realize that they should be modifying a property on the account (which honestly thinking about should be read only...)

@cssuh You said you were going to work on multi-tenant support stuff for the accounts right? This kind of thing should be generic and easily done across multiple extensions - not specific to sql-migration

Reply:
This was an ask from the PM side of the migration team after their accounts ran into some multi tenant issues.

Reply:
However, building an getAccountTenant and running queries on only some tenants should be added to azure core in the future.

Reply:
It would be appreciated if you could spend a little time working with @cssuh on this. Adding in support to at least pass in the tenant into the functions instead of having to modify the account object shouldn't be hard and would benefit other people who also need to do something similar.